### ISSUE TITLE
A mongoose way to handle the createAt and updateAt

### ISSUE THREAD

**Issue by Albert-Gao** [feathersjs/feathers-cli#118]
It's not a bug report, it's an asking which could potentially be a new PR

### Steps to reproduce

Use `feathers generate service`
Choose `Mongoose`
It will generate a Model

No problems here.

The result is something like this:

```javascript
  const post = new mongooseClient.Schema({
    text: {type: String},
    createdAt: { type: Date, default: Date.now },
    updatedAt: { type: Date, default: Date.now }
  });
```

However, for mongoose, it has a built-in way to solve this `time` boilerplate,

```javascript
  const post = new mongooseClient.Schema({
    text: {type: String}
  }, {
    timestamps: true
  });
```
Then the mongoose will handle `createAt` and `updateAt` by itself which we don't need to worry about. It works for `feathers-mongoose` as well.

Shouldn't this be a more `mongoose` way when people choose to use mongoose?
Otherwise, without adding any hook or implementing your own solution, the `updateAt` won't get updated during runtime.

Worth considering? Or the original design is intentional? :)

**Comment by marshallswain** [comment]
@Mr-Binary I would appreciate a PR for this.  I think about this every time I erase those fields and switch to { timestamps: true } (in every model I create).  That's definitely the proper way to handle it, imo.

**Comment by daffl** [comment]
Makes sense. A pull request is definitely very welcome 😄

**Comment by daffl** [comment]
The files in question would be https://github.com/feathersjs/generator-feathers/blob/master/generators/service/templates/model/mongoose-user.js and https://github.com/feathersjs/generator-feathers/blob/master/generators/service/templates/model/mongoose.js

**Comment by Mr-Binary** [comment]
Great!
And thanks for the file locating

PR added: 
[#284](https://github.com/feathersjs/generator-feathers/pull/284)

**Comment by Mr-Binary** [comment]
Something wrong with that test cases, always timeout cause the CI to fail:

[full result](https://travis-ci.org/feathersjs/generator-feathers/jobs/283391927)

```bash
1) generator-feathers feathers:connection rethinkdb:
     Error: Timeout of 300000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
  2) generator-feathers feathers:connection mongodb mongodb:
     Error: Timeout of 300000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
  3) generator-feathers feathers:connection mongodb mongoose:
     Error: Timeout of 300000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
  4) generator-feathers feathers:service mongodb:
     Error: Timeout of 300000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
  5) generator-feathers feathers:service mongoose:
     Error: Timeout of 300000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

**Comment by daffl** [comment]
Closed and released via #284
